services/net/notification/NotificationManager.cs

```diff
@@ -403,7 +403,7 @@ private async Task<string> GenerateNotificationBodyAsync(NotificationModel notif
     private async Task<string> GenerateNotificationAsync(NotificationModel notification, ContentModel content, bool isSubject = false)
     {
         var result = await _engine.CompileRenderStringAsync(
-            notification.Name + "_" + notification.Id + (isSubject ? "_subject" : ""),
+            $"notification_{notification.Id}_{notification.UpdatedOn}{(isSubject ? "_subject" : "")}",
```

Comment:
Renamed cache key so that it will get updated.  This however might cause a memory leak.